nautobot/dcim/templates/dcim/location_migrate_data_to_contact.html

```diff
@@ -0,0 +1,131 @@
+                        <p class="form-control-static">{{ obj.contact_phone | placeholder}}</p>
+                    </div>
+                </div>
+                <div class="form-group">
+                    <label class="col-md-3 control-label required">Source Location Contact Email</label>
+                    <div class="col-md-9">
+                        <p class="form-control-static">{{ obj.contact_email | placeholder}}</p>
+                    </div>
+                </div>
+                {% render_field form.name %}
+                {% render_field form.phone %}
+                {% render_field form.email %}
+                {% render_field form.contact %}
+                {% render_field form.team %}
+                {% render_field form.role %}
+                {% render_field form.status %}
+            </div>
+        </div>
+    </div>
+{% endblock %}
+
+{% block buttons %}
+    <button type="submit" name="_create" class="btn btn-primary">Assign</button>
+    <a href="{{ cancel_return_url }}" class="btn btn-default">Cancel</a>
+{% endblock %}
+
+{% block javascript %}
+<script>
+
+    // action drop down toggle
+    const action = document.getElementById("id_action");
+    function toggle_form_fields() {
+        const action_option = action.value;
+        const name = document.getElementById("id_name");
+        const name_label = document.querySelector("label[for=id_name]");
+        const phone = document.getElementById("id_phone");
+        const email = document.getElementById("id_email");
+        const similar_contacts_label = document.querySelector("label[for=id_contact]");
+        const similar_contacts = document.getElementById("id_contact");
+        const similar_teams_label = document.querySelector("label[for=id_team]");
+        const similar_teams = document.getElementById("id_team");
+
+        // Toggle these form fields when the action matches
+        similar_contacts.toggleAttribute("required", action_option=="use existing contact");
+        similar_teams.toggleAttribute("required", action_option=="use existing team");
+        name.toggleAttribute("disabled", action_option.match(/use existing/));
+        phone.toggleAttribute("disabled", action_option.match(/use existing/));
+        email.toggleAttribute("disabled", action_option.match(/use existing/));
+
+        // Show and hide form fields and toggle form field labels
+        if (action_option === "use existing contact"){
+            similar_contacts.parentElement.parentElement.style.display = "block";
+            if (! similar_contacts_label.className.includes(" required")){
+                similar_contacts_label.className += " required";
+            }
+            similar_teams.parentElement.parentElement.style.display = "none";
+            if (similar_teams_label.className.includes(" required")){
+                similar_teams_label.className = similar_teams_label.className.replace(" required", "");
+            }
```

Comment:
Looks like you can use [`.classList.toggle()`](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList)  to simplify this logic too.